Added sound effects. Issue #2

cosminlupu opened the pull request.

cosminlupu committed:
Added sound effects. Issue #2

samiheikki commented on src/components/SoundToggle.vue:
```diff
@@ -0,0 +1,31 @@
+<template>
+  <p>Sound: <a href="#" v-on:click="onClick"><span v-if="sound">On</span><span v-else>Off</span></a></p>
```
Can this have sound off/on icon instead? Like: https://material.io/icons/#ic_volume_off https://material.io/icons/#ic_volume_up

samiheikki commented on src/App.vue:
```diff
@@ -65,6 +71,24 @@ export default {
+      volume: 0.5
+    })
+    this.correctSound = new Howl({
+      src: ['../static/sounds/correct.mp3', '../static/sounds/correct.ogg'],
```
Correct sound was the only one that I did not like. Can this be changed to something a little less disturbing "correct" sound?

samiheikki commented on src/App.vue:
```diff
@@ -65,6 +71,24 @@ export default {
     })
+
+    // Sound default on
+    let localStorageSound = window.localStorage.getItem('sound')
```
Can you move these to a `initializeSounds` etc method, so `created` method would not become more complex

samiheikki reviewed (commented):
Super thanks to you @cosminlupu! I was really impressed 🎉 I added a couple of comments in there. Please let me know if you want to continue. If not, I can continue the work from this branch. Again, thank you!

cosminlupu committed:
Added some improvements for the sound effects

cosminlupu commented:
Hi @samiheikki,

I've added the required improvements. Regarding the "correct" sound, if the new one i added is not ok, please take over and replace the mp3 and ogg files with some that you like. Another option might be to adjust the `rate` option in the sound initialisation to change it a bit. You can see more about that on the [Howler Repo](https://github.com/goldfire/howler.js#rate-number-10).

samiheikki commented:
Thank you @cosminlupu !!!!!! 🎉 🎉 🎉 🎉 🎉 🎉 🎉
